alibi/explainers/backends/cfrl_tabular.py

```diff
@@ -285,7 +285,7 @@ def generate_condition(X_ohe: np.ndarray,
 
 def sample_numerical(X_hat_num_split: List[np.ndarray],
                      X_ohe_num_split: List[np.ndarray],
-                     C_num_split: List[np.ndarray],
+                     C_num_split: Optional[List[np.ndarray]],
```

Comment:
`None` case is dealt with in the body so it can clearly be `Optional` (as inferred by the type-checker elsewhere in the code).